__tests__/app.test.js

```diff
@@ -60,3 +60,51 @@ describe("/api/topics", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET - status:200 - responds with article's data", () => {
+    test("should return an array of objects", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.result)).toBe(true);
```

Comment:
Unnecessary test, we use a forEach in the next test. However you may want to check the array length is>0 so the assertions inside the forEach are run